pkgs/development/mobile/androidenv/ndk-bundle/default.nix

```
@@ -14,6 +14,9 @@ deployAndroidPackage {
   patchInstructions = lib.optionalString (os == "linux") (''
     patchShebangs .
 
+    # Fix the shebangs of the auto-generated scripts.
+    sed -i 's|#!/bin/bash|#!${pkgs.bash}/bin/bash|' $(pwd)/build/tools/make_standalone_toolchain.py
```

Can we use substituteInPlace here?

```suggestion
    sed -i 's|#!/bin/bash|#!${pkgs.bash}/bin/bash|' build/tools/make_standalone_toolchain.py
```

---

Yes, good point. Forgot about that function. I used ``$(pwd)`` because it was specifically used right below, too. I'll leave it there for consistency replace ``sed`` by substituteInPlace.

Thanks for the hint!

---

> I used $(pwd) because it was specifically used right below, too.

But it is the exact same as if you would start with the directory right away.

---

Exactly. Someone else put it [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/mobile/androidenv/ndk-bundle/default.nix#L20) and I like consistency as I said above. I can
* leave it as is
* remove only "my" ``$(pwd)``
* remove both occurences of ``$(pwd)``

Whatever you feel is best. I'd be happy to get the PR merged.

---

Please just remove it :)

---

See c03f846b46705d282b047a15175203afa3776ed0.